src/components/MuiProvider/index.js

```diff
@@ -11,6 +11,20 @@ export const muiTheme = createTheme({
     borderRadius: 10,
   },
   components: {
+    MuiTypography: {
+      styleOverrides: {
+        root: {
+          color: "#000",
+        },
+      },
+    },
+    MuiContainer: {
+      styleOverrides: {
+        root: {
+          color: "#000",
+        },
+      },
+    },
```

Comment:
Fixed the color issue when using Next.js Link component inside Mui container component.